app/nfsplugin/main.go

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
We can remove `nfsplugin` since this repo will only contain the nfs driver.

Reply:
Same thing here. thanks

Reply:
Done !